p/api/greader.php

```diff
@@ -467,7 +471,7 @@ private static function subscriptionEdit(array $streamNames, array $titles, stri
 	private static function quickadd(string $url) {
 		try {
 			$url = htmlspecialchars($url, ENT_COMPAT, 'UTF-8');
-			if (substr($url, 0, 5) === 'feed/') {
+			if (0 === strpos($url, 'feed/')) {
```

Comment:
That is also not related to this PR, but if anything, the new [`str_starts_with()`](https://php.net/function.str-starts-with) should be used, with a polyfill in `lib_rss.php`.

Reply:
Done